Makefile.am

```diff
@@ -124,8 +124,9 @@ all-local:
 		--build-base $(shell readlink -f $(builddir))/build \
 		--verbose)
 
-if PREFIX_IS_USR
-python_prefix = --root=$(DESTDIR)
+# Fix for GNU/Linux
+if UNAME_IS_DEBIAN
```

Comment:
UPDATED `if`/`fi` to check for Debian specific flag `UNAME_IS_DEBIAN` - which should not interfere with other distributions as the previous implementation using `PREFIX_IS_USR` did.